packages/react/src/components/dropzone.tsx

```diff
@@ -72,11 +72,11 @@ const FilesPreview = ({
           }}
         >
           {/* Hover delete overlay */}
-          <div className="absolute left-0 top-0 flex h-full w-full items-center justify-center rounded-lg bg-red-600/60 text-white opacity-0 group-hover:opacity-100">
+          <div className="absolute left-0 top-0 flex h-full w-full items-center justify-center rounded-lg bg-black/60 text-white opacity-0 group-hover:opacity-100">
```

Comment:
Much better, red makes me feel like I am having a heart attack and black doesn't